### ISSUE TITLE
Angular v6 shows warning about formControlName and ngModel present on the same element

### ISSUE THREAD

**Issue by exequiel09** [Gbuomprisco/ngx-chips#764]
PLEASE MAKE SURE THAT:
- you searched similar issues online (9/10 issues in this repo are solved by googling, so please...do it)
- you provide an online demo I can see without having to replicate your environment
- you help me by debugging your issue, and if you can't, do go on filling out this form

**I'm submitting a ...**  (check one with "x")
```
[X] bug report => search github for a similar issue or PR before submitting
[ ] support request/question

Notice: feature requests will be ignored, submit a PR if you'd like
```

**Current behavior**

No bugs but forward compatibility with Angular v7.

**Expected behavior**

No bugs but forward compatibility with Angular v7.

**Minimal reproduction of the problem with instructions (if applicable)**

Just use Angular v6 above and warning will be shown when the ngx-chips is used in the template.

https://github.com/Gbuomprisco/ngx-chips/blob/f5d218b0f2503aef0cae69a4fee4172e43d40236/modules/components/tag-input-form/tag-input-form.template.html#L11-L12

![Angular v6 Warning](https://user-images.githubusercontent.com/1289472/40284653-1b3da1b8-5cc5-11e8-9f96-a9a5fb3f7bb8.png)

**What do you use to build your app?. Please specify the version**

Angular CLI: 6.0.3

**Angular version:** 

Angular: 6.0.2

**ngx-chips version:**

ngx-chips: 1.9.2

**Browser:** all

**Comment by elessariel** [comment]
I can confirm that. I would also like to see a fix for that.

**Comment by mattqs** [comment]
I was able to fix this warning:

[qs-ngchips](https://www.npmjs.com/package/qs-ngchips)

The changes are in

[tag-input-form.template.html](https://github.com/Quobject/qs-ngchips/blob/master/qs-ngchips-playground/projects/qs-ngchips/src/lib/components/tag-input-form/tag-input-form.template.html)

removing ```[(ngModel)]```

and in 

[tag-input-form.component.ts](https://github.com/Quobject/qs-ngchips/blob/master/qs-ngchips-playground/projects/qs-ngchips/src/lib/components/tag-input-form/tag-input-form.component.ts)

Sorry, I don't have the time to make a pull request. I hope this will help.

**Comment by truantdamian** [comment]
i don´t have these files in my "node-modules"

![image](https://user-images.githubusercontent.com/7271947/42282279-601d4e3a-7f7c-11e8-8104-8d24e28336a9.png)

someone knows why?

**Comment by Gbuomprisco** [comment]
Should be solved with version 1.9.4
